test/monitoring/aws/lambda/metrics.test.ts

```diff
@@ -0,0 +1,47 @@
+import { LambdaMetricFactory } from '../../../../src/monitoring/aws/lambda/metrics';
+
+const DummyTableName = 'DummyTableName';
+
+test('snapshot test: metricInvocations', () => {
+  // GIVEN
+  const unitToTest = new LambdaMetricFactory();
+
+  // WHEN
+  const metric = unitToTest.metricInvocations(DummyTableName);
+
+  // THEN
+  expect(metric).toMatchSnapshot();
+});
+
+test('snapshot test: metricDuration', () => {
+  // GIVEN
+  const unitToTest = new LambdaMetricFactory();
+
+  // WHEN
+  const metric = unitToTest.metricDuration(DummyTableName);
+
+  // THEN
+  expect(metric).toMatchSnapshot();
+});
+
+test('snapshot test: metricErrors', () => {
+  // GIVEN
+  const unitToTest = new LambdaMetricFactory();
+
+  // WHEN
+  const metric = unitToTest.metricErrors(DummyTableName);
+
+  // THEN
+  expect(metric).toMatchSnapshot();
+});
+
+test('snapshot test: metricThrottles', () => {
+  // GIVEN
+  const unitToTest = new LambdaMetricFactory();
+
+  // WHEN
+  const metric = unitToTest.metricThrottles(DummyTableName);
+
+  // THEN
+  expect(metric).toMatchSnapshot();
```

Comment:
Do you think it is better to also add explicit assertions, along with snapshots, for the most important properties? In this case that would probably be the `metricName` and `dimensions`. Thoughts?

Reply:
I do not see any major reason not to. Name and dimensions sounds as a good minimum, the snapshot can take care of the rest (color, labels mainly).